src/app/components/NewUserForm/index.tsx

```diff
@@ -112,8 +114,22 @@ export const NewUserForm = () => {
 
       handleClearFields();
       setKey(+new Date());
+      toast({
+        title: "Sucesso!",
+        description: "Usuário cadastrado com sucesso",
+        classNameContent: "flex flex-row gap-1 pl-2",
+        className: "p-4",
+        variant: "success",
+      });
     } catch (error) {
       console.error("Erro ao cadastrar usuário:", error);
+      toast({
```

Comment:
Será que não conseguimos fazer um esquema de toast.success({}) ao invés de passar como variant?

Reply:
Fiz a alteração no componente  e assim que for aprovado vou modificar no projeto.